### PR TITLE
Create valid Atom feed according to validator.w3.org

### PR DESCRIPTION
**Pull request by tnjd**

### CODE REVIEW

File: src/blog.rs [hunk 1]
```diff
@@ -128,13 +128,14 @@ impl Blog {
     }
 
     pub fn generate_actions<P: AsRef<Path>>(&self, root: P) -> Result<Vec<FsAction>> {
-        let blog_title = "pagong".to_string(); // TODO user-provided
+        let blog_root = "https://tanuj.dev/posts".to_string(); // TODO user-provided
```
Comment (Lonami) [inline src/blog.rs]: lol

File: src/blog.rs [hunk 2]
```diff
@@ -198,10 +199,14 @@ impl Blog {
                 .expect("Escaping HTML in-memory failed");
 
             entries.push(
-                // Additionally, we could add author or category information here
+                // Additionally, we could add category or other extra information here
                 atom::Entry {
                     title: post.title.clone(),
-                    id: post_path.to_string_lossy().to_string(),
+                    id: PathBuf::from(&blog_root)
```
Comment (Lonami) [inline src/blog.rs:206]: Could we add a comment on why to `post_path` alone is not enough?

File: src/blog.rs [hunk 3]
```diff
@@ -261,14 +270,22 @@ impl Blog {
         actions.push(FsAction::WriteFile {
             path: root.as_ref().join("atom.xml").into(),
             content: atom::Feed {
-                title: blog_title.clone(),
-                id: blog_title,
+                title: "BLOG TITLE TODO".into(),
```
Comment (Lonami) [inline src/blog.rs]: Why can't this remain being `blog_title`?

**Review by Lonami — CHANGES_REQUESTED** [review]
I think we should a comment explaining the importance of w3 validty and maybe even a test (not necessarily in this PR, but at least the comment yes).

File: src/blog.rs (hunk 3 above)
Comment (tnjd) [reply src/blog.rs]: good catch ty

File: src/blog.rs (hunk 2 above)
Comment (tnjd) [reply src/blog.rs:206]: Done

File: src/blog.rs (hunk 1 above)
Comment (tnjd) [reply src/blog.rs]: changed to pagong details

**Review by Lonami — APPROVED** [review]

**Pull request merged**